ee/clickhouse/migrations/0025_remove_dollar_from_materialized_columns.py

```diff
@@ -0,0 +1,74 @@
+from functools import lru_cache
+
+import structlog
+from infi.clickhouse_orm import migrations
+
+from ee.clickhouse.client import sync_execute
+from ee.clickhouse.materialized_columns.columns import get_materialized_columns, materialized_column_name
+from posthog.constants import GROUP_TYPES_LIMIT
+from posthog.settings import CLICKHOUSE_CLUSTER, CLICKHOUSE_DATABASE
+
+logger = structlog.get_logger(__name__)
+
+
+def alias_column(table, current_name, new_name):
+    sync_execute(
+        f"""
+        ALTER TABLE {table} ON CLUSTER '{CLICKHOUSE_CLUSTER}'
+        ADD COLUMN IF NOT EXISTS \"{new_name}\"
+        VARCHAR ALIAS \"{current_name}\"
+    """
+    )
+
+    comment = get_column_names(table)[current_name]
+    if len(comment) > 0:
+        comment_column(table, current_name, "")
```

Comment:
If we fail for any reason on the second comment, I guess we would not used a materialized column for some queries. Is it worth switching the order here such that we always maintain at least one materialized column comment (I have no idea what the behaviour here is however, but maybe it just uses the first one it comes across?)

I assume also that it will use the cached materialized column names also, but that's fine because the old name still exists. I like this solution

Reply:
> Is it worth switching the order here such that we always maintain at least one materialized column comment 

I am not sure how code would behave if some comments are duplicated - I think it'll be undeterministic. Losing columns sounds better this way.

You're right in that this migration isn't really rollbackable, but it should be fine as prod is currently the only sharded cluster if we take care there.